django_forest/actions/hooks/views/utils.py

```diff
@@ -26,18 +32,32 @@ def format_enums(self, data):
                 self.handle_enums(field)
 
     def get_hook_result(self, action_name, request):
```

Comment:
I'm not sure about these functions (__flatten_actions__, __get_action__). 
I think it's not mandatory to build a list and reduce it to find the queried action.

Should be better and easier to do something like that:
````python

def _get_action(self, action_name):
  for collection_schema in Collection._registry.values():
    for action in collection_schema.get('actions'):
      name = action['endpoint'].split('/')[-1]
      if name == action_name:
        return action
   raise Exception('Action not found')

def get_hook_result(self, action_name, request):
  queried_action = self._get_action(action_name)
  data = self.handle_action(action, action_name, request)
  self.format_enums(data)
  return data
````
What do you think about this implem ?

Reply:
Yep this looks good -- I've kept the helper method for shared logic.